labs/TrapheusAI/trapheusai_app.py

```diff
@@ -98,6 +101,9 @@ def style_page():
             """
     streamlit.markdown(style, unsafe_allow_html=True)
 
+def showDownloadMessage():
+    streamlit.success("Downloading dataset to system")
```

Comment:
Can we put the message as "Your dataset has been saved"

Reply:
Updated message